packages/orchestration/src/typeGuards.js

```diff
@@ -293,15 +293,32 @@ export const ForwardOptsShape = M.splitRecord(
  * @type {TypedPattern<IBCMsgTransferOptions>}
  * @internal
  */
-export const IBCTransferOptionsShape = M.splitRecord(
-  {},
-  {
-    timeoutTimestamp: M.bigint(),
-    timeoutHeight: {
-      revisionHeight: M.bigint(),
-      revisionNumber: M.bigint(),
+export const IBCTransferOptionsShape = M.or(
```

Comment:
I think this is correct but I do wish we had more expressivity in Patterns

Reply:
- #6115

(should that move to endo?)